### ISSUE TITLE
Spacing for post type and comments

### ISSUE THREAD

**Issue by amongthestones** [jacklenox/susty#12]
I recently found [Delivering WordPress in 7KB](https://blog.jacklenox.com/2018/06/04/delivering-wordpress-in-7kb/) and really like the theme for its low bandwidth and its minimal look. But there are spaces on either side of post types and also the Leave a Comment link. Were these intentional? I was looking to delete them but I haven’t found the line (not much of a coder in spite of the GitHub account).

**Comment by jacklenox** [comment]
@amongthestones Thank you for reporting this. Can you give a bit more detail? I'm not sure I follow exactly what you mean. Could you maybe add a screenshot?

**Comment by amongthestones** [comment]
Here's a screenshot of what I mean. There are two spaces on either side of the post type (in this case, Links), and the leave a comment section.
![screenshot 2019-01-14 at 16 59 58](https://user-images.githubusercontent.com/16293621/51124697-34c6b000-181f-11e9-83fa-4669047e90b3.png)

**Comment by sgclark** [comment]
I too recently adopted the Susty theme and have similar questions about the "Meta" details for posts.  I can't seem to find the PHP file that contains these details.  I too would like to make some edits and changes to these items.  I also wanted to put the "Leave A Comment" link on a different line than the "Tags" and maybe move those "Posted" and "Tagged" meta data details up to the "Posted" and "Author" details at the top of the post.  Anyway, really just curious where the file is that I can make these changes to.

My site is www.sgclark.com if you want to check it out.

Thanks!

**Comment by jacklenox** [comment]
Hi all, so sorry for not replying on this sooner. @amongthestones So yeah, this is a super weird design decision that I made. I don't really know why. :man_shrugging: I kind of wanted them to be slightly button-like, but on reflection it's a bit rubbish! The padding is added here:

https://github.com/jacklenox/susty/blob/7a43aa1a57d646362354178b2842fd6d5fc3c2eb/style.css#L930-L933

@sgclark Nice child theme! It looks as though you solved the issue yourself, but let me know if you have any other questions.

**Comment by sgclark** [comment]
Yes, I did eventually figure it out but what I did does get a little ‘moody’ on different sized devices.  I’ll compare your code above to what I did to see if it behaves better (which I am sure it will).

Thanks, re: my site.  I love how damn quick the base Susty code is when the site loads and even with the additional things I have on my site (all the images/galleries and the mobile nav treatment), it still flies!!

**Comment by sgclark** [comment]
Finally got around to looking at the "Meta Tags" code I'm using compared to what you noted above.  

The weird thing I am seeing was that the font size of the links in the "meta tags" rendered at a smaller size than the 'plain' text in the 'meta tags'.  I used the <code>font-size: inherit</code> css to normalize the links in my CSS to match the size of the plain text but I was wondering if there is some font size control in the master Susty CSS that was controlling that?  I'm also seeing some extra spacing/padding before and after the <code>.cat-links</code> links.  

Not a big deal but curious none the less.

**Comment by HDValentin** [comment]
Link: [https://happy-heidelberg.de/teamblog/](https://happy-heidelberg.de/teamblog/)

I've changed my styles.css to the following to solve the issue:

`
.entry-meta {color: rgb(91, 91, 91);}
.entry-meta a {color: rgb(91, 91, 91);}

.cat-links, .tags-links {
	align-items: left;
}

.cat-links a, .tags-links a, .comments-link a, .edit-link a {
	margin: 0 0 0 .3em;
}
`

In inc/template-tags.php I've added a `<br>`:

`	function susty_wp_entry_footer() {
		// Hide category and tag text for pages.
		if ( 'post' === get_post_type() ) {
		
			/* translators: used between list items, there is a space after the comma */
			$categories_list = get_the_category_list( esc_html__( ', ', 'susty' ) );
			if ( $categories_list ) {
				/* translators: 1: list of categories. */
				printf( '<span class="cat-links">' . esc_html__( 'Kategorie: %1$s', 'susty' ) . '</span>', $categories_list ); // WPCS: XSS OK.
		
			}

			/* translators: used between list items, there is a space after the comma */
			$tags_list = get_the_tag_list( '', esc_html_x( ', ', 'list item separator', 'susty' ) );
			if ( $tags_list ) {
		 
				/* translators: 1: list of tags. */
				printf( '<br><span class="tags-links">' . esc_html__( 'Stichworte: %1$s', 'susty' ) . '</span>', $tags_list ); // WPCS: XSS OK.
		
			}
		}`
